### PR TITLE
Prize qualification baselines

### PR DESCRIPTION
**Pull request by priyakasimbeg**
Add prize qualification baselines for external en self-tuning rulesets. 

@sourabh2k15 Could you double check that the parameter points are correct, they should correspond to internal opt list points 3, 4, 5, 6 and 8. 
Used nadamw with optlist point 8 for self-tuning ruleset.

### CODE REVIEW

File: reference_algorithms/threshold_baselines/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Threshold Baselines
```
Comment (georgedahl) [inline reference_algorithms/threshold_baselines/README.md:1]: Let's call them "Prize qualification baselines"
Try to just spell out what they are in prose and only use that 3 word name in paths and folders and filenames.

Comment (georgedahl) [reply reference_algorithms/threshold_baselines/README.md:1]: We don't want people to confuse them with the qualification workloads/leaderboard, so it is good to also have the word "prize"

File: reference_algorithms/prize_qualification_baselines/external_tuning/tuning_search_space.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
+[
+    {
+        "dropout_rate": 0.0,
+        "label_smoothing": 0.1,
+        "learning_rate": 0.001308209823469072,
+        "one_minus_beta1": 0.02686663061,
+        "beta2": 0.9981232922116359,
```
Comment (sourabh2k15) [inline reference_algorithms/prize_qualification_baselines/external_tuning/tuning_search_space.json:7]: beta1 values are missing, could you copy that over ?

Comment (sourabh2k15) [reply reference_algorithms/prize_qualification_baselines/external_tuning/tuning_search_space.json:7]: nvm, I see one_minus_beta1 values

**Review by sourabh2k15 — APPROVED** [review]

File: reference_algorithms/threshold_baselines/README.md (hunk 1 above)
Comment (priyakasimbeg) [reply reference_algorithms/threshold_baselines/README.md]: Fixed

**Pull request merged**